app/src/main/java/com/activity_sync/renderers/base/AdapteeCollection.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2014 Pedro Vicente Gómez Sánchez.
```

Comment:
Add a comment to the code mentioning from where this library comes from (i.e. URL), add a comment to places which were modified (if reasonable) and explain in comments why was it modified.